### ISSUE TITLE
EFB Failures page documentation for both aircraft

### ISSUE THREAD

**Issue by alepouna** [flybywiresim/docs#1057]
We should create pages for the failures each aircraft has available to document what exactly they will fail and how to recover (or if there is any recovery) 

I'd like to tackle this for the A32NX for starters, so I am going to self assign. I will create a relevant PR later today. 

Before starting to work on it, I would like to know which place would be best to shove this information under? Advanced Guides -> Failures or somewhere else? 

Note to self: Update contributing information for main repo to ensure new system failures from the EFB added to the aircraft are also documented (Needs Documentation tag)

**Comment by frankkopp** [comment]
Normally I'd say there are two parts. 
The EFB part should be in the EFB section. From there there could be links to failure specific Advanced Guides.

**Comment by alepouna** [comment]
> Normally I'd say there are two parts. The EFB part should be in the EFB section. From there there could be links to failure specific Advanced Guides.

Oh yeah good shout. Something like: 

flypad failures page - shows information about how the page works, how to notice active failures etc. with links to the A32NX and A380X available failures pages.

**Comment by frankkopp** [comment]
Exactly. The Advanced Guides could then dive deeper into how to handle these situations. Think "abnormal procedures"
